src/main/java/rs117/hd/HdPlugin.java

```diff
@@ -427,14 +421,15 @@ protected void startUp()
 		configExpandShadowDraw = config.expandShadowDraw();
 		configHdInfernalTexture = config.hdInfernalTexture();
 		configWinterTheme = config.winterTheme();
+		configMaxDynamicLights = config.maxDynamicLights().getValue();
```

Comment:
I'm curious, is copying configuration values really necessary? I know we've inherited this style but I can't imagine that there's much overhead to accessing the config

Reply:
I think we had this discussion before and said that the overhead is probs nothing, 117 did it for stuff that is called all the time but we seem to do it for everything now. So kinda pointless I think

Reply:
Most of them are unnecessary I think, but it might be that one or two actually impact performance for all I know. `configMaxDynamicLights` specifically has to be changed carefully in sync with rendering as well (which I've just fixed).